HGustavs/LenaSYS
# Possibly move php code for fetching codeexamples into relevant php file #15039

Moved fetching of gitcodeexamples from sectioned.php to sectionedservice.php.

Test:

Create a new codeexample.
- choose a template.
- fetch files from github by adding a github url and filepath.
After this there should be insertions in 
- codeexample, fileLink, box database tables.
- gitFiles and gitRepos in metadata2.db

The codeexample should, when pressed should show the file content as usual.
